exo-run/features/steps/steps-given.ls

```diff
@@ -18,4 +19,4 @@ module.exports = ->
                                        cwd: @app-dir,
                                        stdout: dim-console.process.stdout
                                        stderr: dim-console.process.stderr)
-      done!
+      wait 10_000, done
```

Comment:
That won't work because `depends_on` only checks if a container has been started, not if it has properly booted up and is ready to accept connections.